.github/workflows/nightly-builds.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Cache Coursier cache
         uses: coursier/cache-action@v6.2
 
-      - name: sbt akka-cluster-metrics/test
+      - name: sbt pekko-cluster-metrics/test
```

Comment:
this hasn't yet changed in the build.sbt - I have a PR that starts the process of renaming these modules